### PR TITLE
Add-ready-notReady-value-in-Todo-and-create-additional-controller

### PR DESCRIPTION
**Pull request by PalamarchukVladislav**
Add ready status value and create additional controller for Todo entity

### CODE REVIEW

File: src/main/java/com/knighttodo/knighttodo/controller/IsReadyController.java [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+import org.springframework.web.bind.annotation.RestController;
+
+import javax.validation.Valid;
+
+
+@RestController
+@RequestMapping("/ready")
+public class IsReadyController {
+
+    private TodoService todoService;
```
Comment (FedosovMax) [inline src/main/java/com/knighttodo/knighttodo/controller/IsReadyController.java:19]: Here it's better to use final, it's a best practice, I'll tell you about it leter.

File: src/main/java/com/knighttodo/knighttodo/controller/IsReadyController.java [hunk 2]
```diff
@@ -0,0 +1,33 @@
...
+
+    public IsReadyController(TodoService todoService){
+        this.todoService = todoService;
+    }
```
Comment (FedosovMax) [inline src/main/java/com/knighttodo/knighttodo/controller/IsReadyController.java:23]: Better to use @RequiredArgsConstructor lombok annotation above the class.

**Review by FedosovMax — APPROVED** [review]

File: src/main/java/com/knighttodo/knighttodo/controller/IsReadyController.java (hunk 1 above)
Comment (PalamarchukVladislav) [reply src/main/java/com/knighttodo/knighttodo/controller/IsReadyController.java:19]: Okay

File: src/main/java/com/knighttodo/knighttodo/controller/IsReadyController.java (hunk 2 above)
Comment (PalamarchukVladislav) [reply src/main/java/com/knighttodo/knighttodo/controller/IsReadyController.java:23]: Yes, i changed it

**Pull request merged**